codility_training/lessons.lesson02.Arrays.CyclicRotation/khw970421.js

```diff
@@ -0,0 +1,9 @@
+function solution(A, K) {
+    // write your code in JavaScript (Node.js 8.9.4)4
+    let b=[];
+    for(let i=0;i<A.length;i++)
```

Comment:
```suggestion
    for(let i=0, len = A.length;i<len;i++)
```
length 계산에 대한 변수를 만들것!

Reply:
매번 A.length를 구해내는 시간보다 변수써서 하는게 시간이 더빠르게 걸리나보네요